target/linux/ramips/dts/mt7620a.dtsi

```diff
@@ -450,7 +447,7 @@
 		#phy-cells = <0>;
 
 		ralink,sysctl = <&sysc>;
-		resets = <&rstctrl 22 &rstctrl 25>;
+		resets = <&sysc 22 &sysc 25>;
```

Comment:
Seems like it does not matter, and I did not see it mentioned in the reset binding documentation, but would we be cleaner referencing these as a list (if we are changing them anyway)?
```suggestion
		resets = <&sysc 22>, <&sysc 25>;
```

Reply:
Changes applied.
I will take a closer look. Some clocks and resets properties seem useless, perhaps we should drop them directly.